### PR TITLE
Added Utility for normalization of connectors

### PR DESCRIPTION
**Pull request by SalmanMesia**
Added utility to create our own array of attributes from connectors

The attributes array can be used as a custom property in any SDO, that includes the name, value, and type of that attribute

### CODE REVIEW

File: stix_shifter/requirements.txt [hunk 1]
```diff
@@ -12,3 +12,5 @@ xmltodict==0.13.0
 jsonmerge==1.8.0
 colorlog==6.6.0
 stix2-matcher==3.0.0
+ipaddress==1.0.23
```
Comment (mdazam1942) [inline stix_shifter/requirements.txt:15]: ipaddress is part of standard python https://docs.python.org/3.9/library/ipaddress.html

File: stix_shifter_utils/normalization/normalization_helper.py [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+  :return: ip if it is a string which is an ipaddress
+  :return: URI if it is a URL
+  :return: lat_lng if it is a latitude or longitude coordinates
+  '''
+  attribute_type = None
+  if isinstance(attribute, (int, float, complex)):
+    attribute_type = 'number'
+  if isinstance(attribute, (str)):
+    attribute_type = 'string'
+    if validators.url(attribute):
```
Comment (mdazam1942) [inline stix_shifter_utils/normalization/normalization_helper.py:56]: validator library uses a whole bunch of regex underneath to validate. why don't you use `urllib.parse.urlparse`? you won't have to add any extra library to stix-shifter.

File: stix_shifter/requirements.txt (hunk 1 above)
Comment (SalmanMesia) [reply stix_shifter/requirements.txt]: Removed

File: stix_shifter_utils/normalization/normalization_helper.py (hunk 2 above)
Comment (SalmanMesia) [reply stix_shifter_utils/normalization/normalization_helper.py]: Updated to remove validators and use urllib.parse instead

**Review by mdazam1942 — APPROVED** [review]

**Pull request merged**